From 4d9d6686dea19e7fd318c2a09f0c31cf43e9401e Mon Sep 17 00:00:00 2001
From: Xinyu Chen <xinyu.chen@freescale.com>
Date: Wed, 26 Dec 2012 10:50:23 +0800
Subject: [PATCH 126/602] ENGR00238272 imx6 sabresd: disable fade edge effect
 for textview

Disable the Fade edge effect by default in framework, as
it may bring some cost when draw with hardware render enabled.
APK can use android:requiresFadingEdge feature to
enable it in AndroidManifeset.xml.

Signed-off-by: Xinyu Chen <xinyu.chen@freescale.com>
---
 imx6/sabresd/overlay/frameworks/base/core/res/res/values/config.xml | 6 ++++++
 1 file changed, 6 insertions(+)

diff --git a/imx6/sabresd/overlay/frameworks/base/core/res/res/values/config.xml b/imx6/sabresd/overlay/frameworks/base/core/res/res/values/config.xml
index 21fa3f4..edb7b3f 100755
--- a/imx6/sabresd/overlay/frameworks/base/core/res/res/values/config.xml
+++ b/imx6/sabresd/overlay/frameworks/base/core/res/res/values/config.xml
@@ -188,4 +188,10 @@
     -->
     <bool name="config_enableWifiDisplay">true</bool>
 
+    <!-- Disables fading edges when marquee is enabled in TextView.
+         The fading edge effect will bring performance drop when
+	 the hardware rendering is used.
+    -->
+    <bool name="config_ui_enableFadingMarquee">false</bool>
+
 </resources>
-- 
1.8.0

